src/app/components/fees-row/components/fee-estimate-item.tsx

```diff
@@ -30,13 +31,13 @@ export function FeeEstimateItem(props: FeeEstimateItemProps) {
       isInline
       mb="0px !important"
       minWidth="100px"
-      onClick={() => onSelectItem(index)}
+      onClick={() => !disableFeeSelection && onSelectItem(index)}
```

Comment:
I haven't tested this myself in the UI, but why would you even see the selector if selecting the fee is disabled?